components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/BUILD.gn

```diff
@@ -3,9 +3,69 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import("//base/win/embedded_i18n/generate_embedded_i18n.gni")
+import("//brave/components/brave_vpn/common/buildflags/buildflags.gni")
+
+assert(enable_brave_vpn)
+
 source_set("resources") {
   inputs = [
     "resource.h",
     "resources_en.rc",
   ]
+  deps = [ ":strings" ]
+}
+
+generate_embedded_i18n("generate_strings") {
+  visibility = [ ":strings" ]
+  extractor_datafile = "//brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/create_wireguard_string_rc.py"
+  grdfile_folder = "//brave/components/resources"
+
+  xtb_relative_path = "strings"
+  grdfile_name = "brave_components_strings"
+
+  # Exclude languages not supported by Brave
+  # https://github.com/brave/brave-browser/wiki/Strings-and-Localization#information-for-submitting-localization-orders
+  brave_supported_i18_locales = default_embedded_i18_locales - [
```

Comment:
This is not maintainable. If the list of languages changes, this will be out-of-date

Reply:
I think the list of unmaintainable locales should be moved to the config.gni to use from one place, @mkarolin wdyt?